config/templates/metasploit-framework-wrappers/msfwrapper.erb

```diff
@@ -93,10 +88,14 @@ else # Skip initialization if we're root
   fi
 fi
 
+if [ $cmd = "msfconsole" ]; then
 # Uncomment to enable libedit support
-#if [ $cmd = "msfconsole" ]; then
-#    cmd="$cmd -L"
-#fi
+# cmd="$cmd -L"
+  if [ -n "`find $FRAMEWORK/$cmd -mmin +20160`" ]; then
+    (>&2 echo "This copy of metasploit-framework is more than two weeks old.")
```

Comment:
I think it will persist only if you do `exec >&2`. Yeah, we should just delete this.